website/content/api-docs/system/mounts.mdx

```diff
@@ -171,6 +171,12 @@ This endpoint enables a new secrets engine at the given path.
     unversioned plugin that may have been registered, the latest versioned plugin
     registered, or a built-in plugin in that order of precendence.
 
+  - `allowed_managed_keys` `(array: [])` - List of managed key registry entry names
```

Comment:
Fun, guess we missed this.

Reply:
Yeah I've opened https://github.com/hashicorp/vault/pull/23948, so I can backport it.